packages/i18n/locales/en/translation.json

```diff
@@ -159,17 +164,23 @@
     "noNftsSelected": "No NFTs selected.",
     "noValidatorFound": "No validator found.",
     "noVotingPower": "You have not given anyone voting power. Add some members to your DAO.",
+    "notATokenSwapContractAddress": "This contract is not a token swap contract.",
     "notCw20Address": "This contract address is not a CW20 token.",
     "notCw721Address": "This contract address is not a NFT (CW721).",
     "notEnoughForDeposit": "You do not have enough tokens in your balance to pay the proposal deposit",
     "notInActiveSet": "This validator is not in the active set. No rewards will be earned while this is the case.",
+    "notPartyInTokenSwap_dao": "The DAO is not a party in this token swap.",
+    "notPartyInTokenSwap_wallet": "You are not a party in this token swap.",
+    "notYetSentTokenSwap_dao": "The DAO has not yet sent its share of this token swap, so it cannot withdraw.",
```

Comment:
I find this a little confusing. Does it mean that you can't withdraw the counterparty's share (because that share isn't in there yet), or you can't withdraw your share (because the swap is still open)? Depending on which this means, I might suggest a different text.

A main question for me is: what happens if the counterparty never funds? Can you withdraw your share still, or no?